Sources/Mew/Cells+UIKit/TableViewHeaderFooterView+UIKit.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  TableViewHeaderFooterView+UIKit.swift
+//  Mew
+//
+//  Created by Shun Usami on 2019/08/28.
+//  Copyright © 2019 Mercari. All rights reserved.
+//
+
+import UIKit
+
+// MARK: - TableViewHeaderFooterView with UIViewController
+public extension Instantiatable where Self: UIViewController, Self: Injectable {
+    /// Register dequeueable header/footer class for tableView
+    ///
+    /// - Parameter tableView: Parent tableView
+    static func registerAsHeaderFooter(to tableView: UITableView) {
```

Comment:
これも他のメソッドをregisterAsCellとして統一したほうがいいですね、asでいいのかな?

Reply:
Asはなんとなく良さそうです。
```
〔…として〕登録する、〈…を〉〔…として〕登録する
```
https://ejje.weblio.jp/content/register+as


むしろ、 `to tableView`のところをwithかonなどに変えたが良さそうです。
```
register = 登録, and usually takes "with", "on", or "for".
It depends on whether the use is transitive or intransitive (他動詞か自動詞)

◯　register with xxx  
 例 : "If you register (your information) with our company, you can be assured that it will be kept safe."  (A社に登録)

◯　register on xxx
 例 : "I registered (for a class) on their website."  (website で～～の登録を)

◯　register in.  (他動詞みたいに扱う)
例　"I registered (for ～～) in March."  (３月に(なんとかの)登録をしました

◯　register for～～ (～～の登録をする)
例 "I registered for the test"

◯　register to～～ ((【to 動詞】～～をするのを登録する)
例 "I registered to take the test"
```
https://hinative.com/ja/questions/234384

Reply:
Asについてはdequeueの時に省略してるので、省略してもいいかもですね。
どちらかに統一したほうがいい気がしました。

Asあり
```
registerCell(on:)
registerHeaderFooter(on:)
dequeueCell(from:)
dequeueHeaderFooter(from:)
```

Asなし
```
registerAsCell(on:)
registerAsHeaderFooter(on:)
dequeueAsCell(from:)
dequeueAsHeaderFooter(from:)
```